src/index.js

```diff
@@ -1,6 +1,337 @@
 import './style.css';
+import { Service, StorageService } from './js/Task.js';
 
-console.log({
-  a: '2',
-  b: '3',
-});
+const service = new Service();
```

Comment:
I would remove all the logic from here and delegate it to small specific area files.